android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/scenario/LocalWebSocketServerScenario.java

```diff
@@ -21,15 +23,30 @@ public class LocalWebSocketServerScenario extends Scenario {
     private final LocalWebSocketServer mWebSocketServer;
     private State mState = State.NOT_STARTED;
 
+    private PowerConfigProvider mPowerManager;
+
     public LocalWebSocketServerScenario(@NonNull Context context,
                                         @NonNull MobileWalletAdapterConfig mobileWalletAdapterConfig,
                                         @NonNull AuthIssuerConfig authIssuerConfig,
-                                        @NonNull Callbacks callbacks,
+                                        @NonNull Scenario.Callbacks callbacks,
                                         @NonNull byte[] associationPublicKey,
                                         @WebSocketsTransportContract.LocalPortRange int port) {
+        this(context, mobileWalletAdapterConfig, authIssuerConfig, callbacks,
+                associationPublicKey, port, new DevicePowerConfigProvider(context));
+    }
+
+    public LocalWebSocketServerScenario(@NonNull Context context,
```

Comment:
Similar to comment on `PowerConfigProvider`, I'd make this constructor package-scoped, to limit it's availability to unit tests defined in the same package.